90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -1136,14 +1137,112 @@ be_is_locked() {
   return 1
 }
 
+# arg1: BE key source
+# prints: value of org.zfsbootmenu:keysource for the BE, if defined
+# returns: 0 iff the value is defined and not empty
+
+be_keysource() {
```

Comment:
Can we add a check here to validate that the value of `org.zfsbootmenu:keysource` is actually a filesystem present on the machine, and otherwise throw a `zerror` / print nothing / return 1 ?